src/models/user.model.js

```diff
@@ -22,6 +22,26 @@ const User = sequelize.define('users', {
     type: Sequelize.STRING,
     allowNull: false,
   },
+  dob: {
+    type: DataTypes.DATEONLY,
+    allowNull: true,
+  },
+  lang: {
+    type: Sequelize.STRING,
+    allowNull: true,
+  },
+  currency: {
+    type: Sequelize.STRING,
+    allowNull: true,
+  },
+  gender: {
+    type: Sequelize.STRING,
+    allowNull: true,
+  },
+  placeofliving: {
+    type: Sequelize.STRING,
+    allowNull: true,
+  },
```

Comment:
Check the story. You might need more info for payment details for a user profile

Reply:
- About renaming ``placeofliving`` , is not needed, no problem with it, unless you suggest a better name;
- About payment details, I will created it in different table that references users table via users(id), but here is an issue @didasmbalanya ,  this is my task. if you agree with me that I am to do it this this suggestion of creating different table on criteria 2., I will need some more details such as payment details parameters(prices, addedItems, totalPrice, etc). This means that criteria number two would not go with this PR, I would doit later after @Edcreation id done with ``adding items to cart``, and me: @Kress2000  also finished ``removing items to cart`` and ``viewing added items``
![image](https://user-images.githubusercontent.com/91186046/228567992-0cecd1f4-d557-474b-8ebc-0de3c95e28c5.png)

Reply:
but if it is only billing adress, means, it only to add new few columns such as ( account number, telnumber, etc, )